experimental/dds/tree2/src/domains/json/jsonDomainSchema.ts

```diff
@@ -3,6 +3,13 @@
  * Licensed under the MIT License.
  */
 
+// Adding this unused import makes the generated d.ts file produced by TypeScript stop breaking API-Extractor's rollup generation.
+// Without this import, TypeScript generates inline `import("../..")` statements in the d.ts file,
+// which API-Extractor leaves as is when generating the rollup, leaving them pointing at the wrong directory.
+// TODO: Understand and/or remove the need for this workaround.
+// eslint-disable-next-line @typescript-eslint/no-unused-vars, unused-imports/no-unused-imports
+import { ValueSchema } from "../../core";
```

Comment:
https://github.com/microsoft/FluidFramework/pull/17840 is tested across this PR and the error was fixed.